docs/install/install-server/configure-postgres.inc

```diff
@@ -1,11 +1,18 @@
 .. _install-step-2:
 
-Step 2: Install PostgreSQL 10
+Step 2: Install PostgreSQL 13
 -----------------------------
 
-PostgreSQL 10 can be installed by following the `installation guide <https://www.postgresql.org/download/>`_ for your
+PostgreSQL 13 can be installed by following the `installation guide <https://www.postgresql.org/download/>`_ for your
```

Comment:
so we use the packages from postgres, not from RHEL? 

(this is different for ISO I presume?)

Reply:
Indeed. For ISO we use PostgreSQL from the Red Hat Software Collections.